yarn.lock

```diff
@@ -3834,6 +3834,21 @@
     "@typescript-eslint/types" "5.45.1"
     eslint-visitor-keys "^3.3.0"
 
+"@unimodules/core@*":
```

Comment:
As per comment referred by link ... Do we need to remove it from here then ? 

https://github.com/Sphereon-Opensource/ssi-mobile-wallet/pull/23#discussion_r1087732851